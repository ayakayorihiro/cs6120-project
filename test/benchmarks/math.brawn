# ARGS: numbers.tsv
BEGIN { getline; }
{ N[$1]++; SUM[$1]+=$2; }
END {
    OFS="\t"
    for (k in N) {
        print k, ((SUM[k])/N[k]);
    }
}
