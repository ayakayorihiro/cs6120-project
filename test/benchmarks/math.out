burger	4.78854
pizza	4.68403
soda	4.63009
